src/handwritten/_lib.py

```diff
@@ -148,39 +152,67 @@ def task_handle(self) -> type:
     @property
     def cal_handle(self) -> type:
         return CalHandle
-
+    
+    @property
+    def encoding(self):
+        if self._encoding is None:
+            self._import_lib()
+        return self._encoding
+        
     def _import_lib(self):
         """
         Determines the location of and loads the NI-DAQmx CAI DLL.
         """
         self._windll = None
         self._cdll = None
+        self._encoding = None
 
         windll = None
         cdll = None
-
-        if sys.platform.startswith('win') or sys.platform.startswith('cli'):
-            try:
-                if 'iron' in platform.python_implementation().lower():
+        encoding= None
+
+        # Feature Toggle to load nicaiu.dll or nicai_utf8.dll
+        # The Feature Toggle can be set in the .env file
+        load_dotenv()
+        nicaiu_dll_enabled = os.getenv('NICAIU_DLL_ENABLED') == 'true'
```

Comment:
Let's prefix the variable with `NIDAQMX_PYTHON` to make its origin/purpose clear.

Runtime feature toggles and `.env` variables don't necessarily have to be Booleans. 

Suggestion:
- Rename to `NIDAQMX_PYTHON_C_LIBRARY`
- Valid options:
  - Windows: `nicaiu`, `nicai_utf8`
  - Linux: `nidaqmx`
- If the variable is set, load that DLL with no fallback.

Reply:
@bkeryan I have made the necessary changes, however I didn't set `nidaqmx` as an option since we will be loading `nidaqmx` regardless the variable is set or not when running on Linux.

Reply:
ok, that's fine